### PR TITLE
LEAN-2237:FII-210 - Remove designations from the File inspector - Imp…

### PR DESCRIPTION
**Pull request by mvidtman**
…lementation

### CODE REVIEW

File: src/components/FileManager/UploadFileArea.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ import { NativeTypes } from 'react-dnd-html5-backend'
 import styled, { css } from 'styled-components'
 
 import { Upload } from './FileManager'
-import { Action, actions } from './FileSectionState'
+import { Action, actions, State } from './FileSectionState'
```
Comment (jirikaiser-atypon) [inline src/components/FileManager/UploadFileArea.tsx:22]: Looks like State is imported, but it's never used.

File: src/components/FileManager/FilesSection.tsx [hunk 2]
```diff
@@ -49,19 +48,13 @@ export const FilesSection: React.FC<{
...
-        state.uploadedFile,
-        getDesignationName(state.selectDesignation)
-      )
-  }
   const can = useContext(PermissionsContext)
-
+  const EMPTY_DESIGNATION = ''
+  useEffect(() => {
+    dispatch(actions.HANDLE_UPLOAD_ACTION())
+    state.uploadedFile && handleUpload(state.uploadedFile, EMPTY_DESIGNATION)
```
Comment (jirikaiser-atypon) [inline src/components/FileManager/FilesSection.tsx:56]: Can't we remove the designation completely instead of sending an empty string?

**Review by jirikaiser-atypon — CHANGES_REQUESTED** [review]

File: src/components/FileManager/UploadFileArea.tsx (hunk 1 above)
Comment (mvidtman) [reply src/components/FileManager/UploadFileArea.tsx]: State removed from import....

File: src/components/FileManager/FilesSection.tsx (hunk 2 above)
Comment (mvidtman) [reply src/components/FileManager/FilesSection.tsx]: Done...

**Review by jirikaiser-atypon — APPROVED** [review]

**Pull request merged**